contrib_bots/lib/joker.py

```diff
@@ -0,0 +1,52 @@
+# pip install schedule is necessary
+import schedule
+import time
```

Comment:
Could we reorganize the import statements from the shortest on top and the longest down?